Sprint-1/exercises/decimal.js

```diff
@@ -7,3 +7,13 @@ const num = 56.5678;
 // Create a variable called roundedNum and assign to it an expression that evaluates to 57 ( num rounded to the nearest whole number )
 
 // Log your variables to the console to check your answers
+
+wholeNumberPart = Math.trunc(num)
+console.log(wholeNumberPart)
+
+decimalPart = (Math.floor(num).toFixed(4))
+console.log(decimalPart)
```

Comment:
Have you run your code once to see what output it produces? One of the output value does not match the expected value.